.github/workflows/development-e2e-tests.yml

```diff
@@ -0,0 +1,13 @@
+name: Comment-based E2E test
+
+on:
+  issue_comment:
+    types: [created, edited]
+
+jobs:
+  e2e:
+    runs-on: ubuntu-18.04
+    if: github.event.comment.body == 'run e2e'
```

Comment:
Given that GitHub Actions doesn't have manual jobs per se, this is a manual way of triggering them, that doesn't involve sending requests or anything similar.